gpoa/util/rpm.py

```diff
@@ -36,9 +45,19 @@ def is_rpm_installed(rpm_name):
 class Package:
 
     def __init__(self, package_name):
-        self.__install_command = ['/usr/bin/apt-get', '-y', 'install']
-        self.__remove_command = ['/usr/bin/apt-get', '-y', 'remove']
-        self.__reinstall_command = ['/usr/bin/apt-get', '-y', 'reinstall']
+        distributive=getDistributiveVendor()
```

Comment:
I suggest to use `distribution` as a correct variable name.

Reply:
> I suggest to use `distribution` as a correct variable name.

That's what I thought too when I watched the patches.